nav2_behavior_tree/plugins/condition/is_battery_low_condition.cpp

```diff
@@ -29,6 +29,7 @@ IsBatteryLowCondition::IsBatteryLowCondition(
   is_voltage_(false),
   is_battery_low_(false)
 {
+  createROSInterfaces();
```

Comment:
We may retrieve the value from the ports before calling createROSInterfaces().
The intention is to create ROS interfaces to listen to messages before tick() is executed. This ensures that battery information is ready when tick() is called at the first time. However, we still want to use the topic name defined in the BT port.
Otherwise, the default value, battery_topic_("/battery_status"), will be used.

```suggestion
  getInput("min_battery", min_battery_);
  getInput("battery_topic", battery_topic_);
  getInput("is_voltage", is_voltage_);
  createROSInterfaces();
```

Reply:
You're right.
I'll modify it so that the BT port topic name can be used.